src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -68,19 +79,21 @@ public boolean equals(Object other) {
         UserPrefs o = (UserPrefs) other;
 
         return guiSettings.equals(o.guiSettings)
-                && addressBookFilePath.equals(o.addressBookFilePath);
+                && addressBookFilePath.equals(o.addressBookFilePath)
+                && hotelFilePath.equals(o.hotelFilePath);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(guiSettings, addressBookFilePath);
+        return Objects.hash(guiSettings, addressBookFilePath, hotelFilePath);
     }
 
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder();
         sb.append("Gui Settings : " + guiSettings);
         sb.append("\nLocal data file location : " + addressBookFilePath);
+        sb.append("\nLocal data file location 2 : " + hotelFilePath);
```

Comment:
Could we change this 2 lines. e.g. sb.append("\nAddress Book File Path : " + addressBookFilePath);